code/_onclick/item_attack.dm

```diff
@@ -119,6 +119,8 @@
  * See: [/obj/item/proc/melee_attack_chain]
  */
 /atom/proc/attackby(obj/item/attacking_item, mob/user, params)
+	if(atom_storage && atom_storage.attempt_insert(attacking_item, user))
```

Comment:
any reason to not do atom_storage?.attempt_insert for these?

Reply:
no reason in particular, good idea